
package com.divanov.mathwebservice.dto;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 3.5.3
 * 2023-02-19T23:32:06.563+03:00
 * Generated source version: 3.5.3
 */

@WebFault(name = "quadraticEducationFault", targetNamespace = "http://math.ws.divanov")
public class QuadraticEducationFault_Exception extends Exception {

    private com.divanov.mathwebservice.dto.QuadraticEducationFault faultInfo;

    public QuadraticEducationFault_Exception() {
        super();
    }

    public QuadraticEducationFault_Exception(String message) {
        super(message);
    }

    public QuadraticEducationFault_Exception(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public QuadraticEducationFault_Exception(String message, com.divanov.mathwebservice.dto.QuadraticEducationFault quadraticEducationFault) {
        super(message);
        this.faultInfo = quadraticEducationFault;
    }

    public QuadraticEducationFault_Exception(String message, com.divanov.mathwebservice.dto.QuadraticEducationFault quadraticEducationFault, java.lang.Throwable cause) {
        super(message, cause);
        this.faultInfo = quadraticEducationFault;
    }

    public com.divanov.mathwebservice.dto.QuadraticEducationFault getFaultInfo() {
        return this.faultInfo;
    }
}
